src/library/scala/collection/mutable/PriorityQueue.scala

```diff
@@ -328,6 +328,22 @@ sealed class PriorityQueue[A](implicit val ord: Ordering[A])
     pq
   }
 
+  override def copyToArray[B >: A](xs: Array[B], start: Int): Int = {
+    val copied = IterableOnce.elemsToCopyToArray(length, xs.length, start, length)
```

Comment:
arguably this ought to just call/forward to `copyToArray(xs, start, length)`

Reply:
👍